tests/test_jobs.py

```diff
@@ -523,8 +524,7 @@ def test_recommend_matches(self):
         # Test
         expected = {
             'results': {
-                self.john_smith.uuid: sorted([self.john_smith.uuid,
```

Comment:
This is an example of what I've explained in the text of [this commit](https://github.com/chaoss/grimoirelab-sortinghat/pull/312/commits/7cb6238c7643189cf125b81beab91eaa4efd652e).